google-cloud-pubsub/lib/google/cloud/pubsub/v1/doc/google/pubsub/v1/pubsub.rb

```diff
@@ -477,13 +549,27 @@ class UpdateSnapshotRequest; end
       #     For example, consider a subscription whose oldest unacked message is 3 days
       #     old. If a snapshot is created from this subscription, the snapshot -- which
       #     will always capture this 3-day-old backlog as long as the snapshot
-      #     exists -- will expire in 4 days.
+      #     exists -- will expire in 4 days. The service will refuse to create a
+      #     snapshot that would expire in less than 1 hour after creation.
       # @!attribute [rw] labels
       #   @return [Hash{String => String}]
       #     User labels.
       class Snapshot; end
 
-      # Request for the +ListSnapshots+ method.
+      # Request for the GetSnapshot method.<br><br>
```

Comment:
It is strange to see HTML in the documentation. Is it set to output Markdown? I notice that it is using + where I would expect `.

Reply:
I noticed that too. The HTML is present in the source proto. I'm actually not sure what the style guidelines are for proto documentation formatting (and I'll try to track that down), but that would be out of scope here.